### PR TITLE
[Config] Golden gamemode probability proposal

### PR DESCRIPTION
**Pull request by Cenrus**
Increases the likelyhood of other modes running during high population hours on BeeStation Golden, this is a proposal and these numbers can be changed.

## Why it's good for the Game

People don't like rolling heretics/traitors/changelings on 40 pop Bee Golden, as these modes are run every other round. By increasing the chance of other modes, we should see more varied roundtypes.

Traitorbro and ling have been deactivated above 30 players

## Changelog

:cl:
config: Increased chance of Golden running team based antagonists
/:cl:

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -96,10 +96,12 @@ MASTER_MODE secret
 ## Set to 0 to disable that mode.
 
 PROBABILITY TRAITOR 9
-PROBABILITY CULT 6
-PROBABILITY DYNAMIC 6
+PROBABILITY CULT 9
+PROBABILITY REVOLUTION 8
+PROBABILITY CLOCKCULT 8
```
Comment (ghost) [inline config/game_options.txt]: Clockcult 8? Holy shit.

**Comment by NickEmbased** [comment]
LRP :pensive:

File: config/game_options.txt (hunk 1 above)
Comment (ike709) [reply config/game_options.txt]: Clockcult 8 is _definitely_ not going to get in.

Comment (Cenrus) [reply config/game_options.txt]: Bumping it down to 4, or should I go lower?

**Comment by ivanmixo** [comment]
Good idea. It's such a disappointment for us to finally get good pop and then get heretics for the 10th time today.

File: config/game_options.txt (hunk 1 above)
Comment (ghost) [reply config/game_options.txt]: Clockcult is a game mode that is enjoyable in small numbers, setting it to 4 will still quadruple it.

**Comment by Sinestia** [comment]
I don't like some specifics but the spirit is kind of nice

Here's my recommendations:
I'd recommend increasing incursion's odds (actually, I'd love to see this on sage too)
In my experience adminning incursion rounds, people seem to enjoy it a lot.  

Revolution lower, I think it's good as a 4 because it's a hellish gamemode. Very fun sparingly, it shouldn't be very common.

**Comment by ivanmixo** [comment]
> I don't like some specifics but the spirit is kind of nice
> 
> Here's my recommendations:
> I'd recommend increasing incursion's odds (actually, I'd love to see this on sage too)
> In my experience adminning incursion rounds, people seem to enjoy it a lot.
> 
> Revolution lower, I think it's good as a 4 because it's a hellish gamemode. Very fun sparingly, it shouldn't be very common.

Lrp main here, I haven't seen revs on golden in a couple of months. Hell, I've seen clockies more than revs. Those rounds barely happen. I think it would be good to see them more.

**Comment by PowerfulBacon** [comment]
Should just lower traitor / low pop gamemodes and up dynamic. Otherwise it will be cult very often

**Comment by Crossedfall** [comment]
If the goal is to adjust for population, why not adjust the minimum/maximum populations? This doesn't really prevent those modes from showing up on lower pop, which seems like the underlying goal here

**Comment by ivanmixo** [comment]
> If the goal is to adjust for population, why not adjust the minimum/maximum populations? This doesn't really prevent those modes from showing up on lower pop, which seems like the underlying goal here

I am pretty sure the goal is to see conversion type antags more often when we actually have the pop for them.

**Comment by Crossedfall** [comment]
> 
> 
> > If the goal is to adjust for population, why not adjust the minimum/maximum populations? This doesn't really prevent those modes from showing up on lower pop, which seems like the underlying goal here
> 
> I am pretty sure the goal is to see conversion type antags more often when we actually have the pop for them.

So... Adjusting for population then? 😄

**Comment by Cenrus** [comment]
I don't want to eliminate the chances of low population gamemodes during high population times, I just want the other modes to roll more often.

**Review by utilbird — COMMENTED** [review]
Making modes like cult just as/more common than modes like traitor will just make people get sick of them instead. This PR relies on relatively consistent lowpop to have the desired effect which isn't really the case imo

With that being said, I think increasing the frequency of chaotic rounds like cult isn't necessarily a bad thing. It's just the increase is a bit too high at the moment, in my opinion.

**Comment by REALALFREDCOLLINS** [comment]
hop on deep rock galactic

**Comment by Cartlord** [comment]
Honestly, I think gamemode probabilities can use a rework overall, especially considering lowpop BB exists and can lead to three antags in a round of 6 or 10 people.

**Comment by Cartlord** [comment]
> 
> 
> Making modes like cult just as/more common than modes like traitor will just make people get sick of them instead. This PR relies on relatively consistent lowpop to have the desired effect which isn't really the case imo
> 
> With that being said, I think increasing the frequency of chaotic rounds like cult isn't necessarily a bad thing. It's just the increase is a bit too high at the moment, in my opinion.

IDK about you, but in my experience Golden gets lowpop and nopop every night.

**Pull request closed without merging**